configs/fedramp-stage/roles/compliance.json

```
@@ -2,14 +2,63 @@
   "roles": [
     {
       "name": "Compliance administrator",
-      "description": "Perform any available operation against any Compliance resource.",
+      "description": "A Compliance role that grants full access to any Compliance resource.",
       "system": true,
-      "platform_default": true,
-      "version": 8,
+      "version": 9,
       "access": [
         {
           "permission": "compliance:*:*"
         },
+        {
+          "permission": "inventory:*:read"
+        },
+        {
+          "permission": "remediations:remediation:read"
+        },
+        {
+          "permission": "remediations:remediation:write"
+        }
+      ]
+    },
+    {
+      "name": "Compliance reporter",
+      "description": "A Compliance role that grants read access to any Compliance resource.",
```

Based on your comment that this role should allow exports, would it be beneficial to have the description here express that?
```suggestion
      "description": "A Compliance role that grants read and export access to any Compliance resource.",
```

Just a suggestion for that or something similar if it would be beneficial.

---

I like that, thank you!